modeles/notes/MODELS.md

```diff
@@ -0,0 +1,58 @@
+# Models
+
+Here the description of some of the models I found and why are they interesting in the objective to predict affluence for recycling center.
+
+## [Bayesian method](https://en.wikipedia.org/wiki/Bayesian_inference)
+
+The Bayesian method is a way to learn probabilities from patterns, and to gradually improve the quality of the statistics as we have more data.
+
+This can be an interesting approach as we have better data to predict affluence over time ([I mentioned here including the previous hour's affluence improved the prediction](PROJECT.md)); however as people won't need to throw their waste because a lot of people also needed to at the beginning of the day, I think we can find a better model that doesn't only matter on the previous affluence.
+
```

Comment:
You should ask Du but I think with bayesian models you can add contextual features like meteo etc.